### PR TITLE
11.0.1-prerelease.0

### PR DESCRIPTION
**Pull request by ronilan**
Prerelease run with accumulated documentation updates.

Note: 

We've decided that "packages maintainers may release as many prerelease versions as they deem appropriate" without need for SolarWinds approval.

Merging this pull request does not trigger the prerelease.
The trigger is by git tag push that will come after all workflows pass..

Future prereleases will not contain such message.

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -210,6 +210,8 @@ The script will:
 
 To start fresh on the dev repo run `npm run dev:repo:reset` again.
 
+When running a [Release](#release---push-version-tag) process with on the dev repo, the package will be published to https://www.npmjs.com/package/@appoptics/apm-bindings-dev. It should be [unpublished](https://docs.npmjs.com/unpublishing-packages-from-the-registry) as soon as possible. Note that because the package is scoped to the organization, the organization admin must temporarily reassign this package to just the dev-internal team; this team has a single member, which is one of the requirements for unpublishing per https://docs.npmjs.com/policies/unpublish#packages-published-more-than-72-hours-ago.
```
Comment (cheempz) [inline .github/README.md:213]: Nit: extra `with` in the sentence "process with on the dev repo"

File: .github/README.md [hunk 2]
```diff
@@ -245,7 +247,7 @@ manual ──────────► └────────────
   - Build the code pushed on a default image. (`node` image from docker hub).
   - Run the tests against the build.
 * Workflow confirms code is not "broken".
-* Manual trigger supported. Enables to select image from docker hub or local files (e.g. `erbium-buster-slim`) to build code on.
+* Manual trigger supported. Enables to node version.
```
Comment (cheempz) [inline .github/README.md:250]: Nit: should be "Enables selecting the node version"?

**Review by cheempz — APPROVED** [review]
LGTM pending some wording nits, thanks @ronilan!

File: .github/README.md (hunk 2 above)
Comment (ronilan) [reply .github/README.md]: Fixed.

File: .github/README.md (hunk 1 above)
Comment (ronilan) [reply .github/README.md]: Fixed.

**Review by cheempz — APPROVED** [review]
👍

**Pull request merged**